exporter/datadogexporter/factory.go

```diff
@@ -53,6 +53,12 @@ var metricExportNativeClientFeatureGate = featuregate.GlobalRegistry().MustRegis
 	featuregate.WithRegisterDescription("When enabled, metric export in datadogexporter uses native Datadog client APIs instead of Zorkian APIs."),
 )
 
+var metricRemappingEnabledFeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"exporter.datadogexporter.metricremappingenabled",
+	featuregate.StageBeta,
+	featuregate.WithRegisterDescription("When enabled the Datadog Exporter remaps Opentelemetry metrics to Datadog semantics."),
```

Comment:
I think this would be clearer:
```suggestion
	featuregate.WithRegisterDescription("When enabled the Datadog Exporter remaps OpenTelemetry semantic conventions to Datadog semantic conventions."),
```

Better yet if we have a public documentation link to what this mapping does exactly